### DISCUSSION TITLE
Does super-productivity use Pulseaudio or Pipewire?

### DISCUSSION THREAD

**Discussion by ghost** [super-productivity/super-productivity#3776] [Q&A] [answered]
It would be nice if it used pipewire, that is why I am curious.

**Comment by johannesjo** [comment]
Super Productivity uses whatever ships with electron-builder.

**Reply by ghost** [reply, marked as answer]
It seems like it uses Pipewire by default.
https://github.com/electron/electron/issues/29115#issuecomment-840068914
